client/src/views/Explore.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
+import {View, Text, Button, StyleSheet} from 'react-native';
+import {goToFirstScreenInStack, navigateTo} from '../helpers/NavigateHelper';
+
+class Explore extends Component {
+  render() {
+    const {navigation} = this.props;
```

Comment:
should we check navigation not null?

Reply:
by using propTypes ? good suggestion

Reply:
whenever a component is wrapped inside Navigator Container, it will have a navigation object, but it's good to have a prop types check here, will implement now.

Reply:
that is what I meant there

Reply:
since we are using Flow, we should have static propTypes as checker, reference here:

https://flow.org/en/docs/react/components/